apps/public-docsite-v9/src/Concepts/Migration/FromV8/ComponentMapping.stories.mdx

```diff
@@ -55,37 +55,37 @@ Check out the latest schedule information on the [v9 Component Roadmap](https://
 | Layer                  | [Portal](/docs/components-portal--default)                                                                                                   |
 | Link                   | [Link](/docs/components-link--default)                                                                                                       |
 | List                   | _[In Progress](https://github.com/orgs/microsoft/projects/786/views/1?pane=issue&itemId=24404181)_                                           |
-| MessageBar             | _[In Progress](https://github.com/orgs/microsoft/projects/786/views/1?pane=issue&itemId=24402474)_                                           |
+| MessageBar             | [MessageBar](/docs/components-messagebar--default)                                                                                           |
 | Modal                  | [Dialog](/docs/components-dialog--default)                                                                                                   |
 | Nav                    | _[In Progress](https://github.com/orgs/microsoft/projects/786/views/1?pane=issue&itemId=24403433)_                                           |
-| OverflowSet            | priority-overflow                                                                                                                            |
+| OverflowSet            | [Dialog](/docs/components-overflow--default)                                                                                                 |
 | Overlay                | [Portal](/docs/components-portal--default)                                                                                                   |
-| Panel                  | _[In Progess](https://github.com/orgs/microsoft/projects/786/views/1?pane=issue&itemId=24402572)_                                            |
+| Panel                  | [Drawer](/docs/components-drawer--default)                                                                                                   |
 | Popup                  | [Dialog](/docs/components-dialog--default)                                                                                                   |
 | PrimaryButton          | [Button](/docs/components-button-button--default)                                                                                            |
 | PeoplePicker           |                                                                                                                                              |
```

Comment:
@tudorpopams - PeoplePicker is now TagPicker. Let's make a note of it here, I recommend. For Example: `PeoplePicker is now `[TagPicker](https://react.fluentui.dev/?path=/docs/components-tagpicker--default)

Reply:
Updated.